packages/deployer/index.js

```diff
@@ -49,13 +49,21 @@ class Deployer extends Deployment {
     return this.queueOrExec(link(library, destinations, this));
   }
 
-  deploy() {
+  deploy1() {
     const args = Array.prototype.slice.call(arguments);
     const contract = args.shift();
 
     return this.queueOrExec(this.executeDeployment(contract, args, this));
   }
 
+  async deploy() {
+    const args = Array.prototype.slice.call(arguments);
+    const contract = args.shift();
```

Comment:
``` suggestion
  async deploy(contract, ...args) {
```
[Rest parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters) didn't exist when this was originally written. Let's use it if we can now.